src/mapping/models.py

```diff
@@ -12,3 +12,18 @@ def save(self, **kwargs):
         if AreaOfInterest.objects.count() != 0:
             AreaOfInterest.objects.all().delete()
         super().save(**kwargs)  # Call the "real" save() method.
+
+
+class MappingRoute(models.Model):
+    # fields
+    # Stored list of points as arbitrary length JSON field to be consistent with area of interest
+    # altitude is how high the drone needs to be from ground when taking these pictures.
+    points_on_route = models.JSONField()
+    altitude = models.FloatField()
+
+    # methods
+    def save(self, **kwargs):
```

Comment:
If this is a Singleton, please add that to the class name. Not too sure if a Singleton design pattern is necessary though, we can easily support two mapping areas that we do sequentially

Reply:
^